applications/ShapeOptimizationApplication/tests/algorithm_steepest_descent_test/run_test.py

```diff
@@ -25,14 +25,14 @@
 # Test results and clean directory
 # =======================================================================================================
 output_directory = parameters["optimization_settings"]["output"]["output_directory"].GetString()
-response_log_filename = parameters["optimization_settings"]["output"]["response_log_filename"].GetString() + ".csv"
+optimization_log_filename = parameters["optimization_settings"]["output"]["optimization_log_filename"].GetString() + ".csv"
```

Comment:
Are the default settings added to this "parameters" variable? Or how can you get it from the Parameters object if they are not defined in the json file?